src/plugins/auto/auto_executable_network.cpp

```diff
@@ -45,8 +45,19 @@ IE::Parameter AutoExecutableNetwork::GetMetric(const std::string& name) const {
             ov::PropertyName{ov::optimal_number_of_infer_requests.name(), ov::PropertyMutability::RO},
             ov::PropertyName{ov::hint::model_priority.name(), ov::PropertyMutability::RO},
             ov::PropertyName{ov::device::priorities.name(), ov::PropertyMutability::RO},
-            ov::PropertyName{ov::execution_devices.name(), ov::PropertyMutability::RO}
-        };
+            ov::PropertyName{ov::execution_devices.name(), ov::PropertyMutability::RO}};
+    } else if (name == ov::hint::performance_mode) {
+        auto value = _autoSContext->_performanceHint;
+        if (!_autoSContext->_core->isNewAPI())
+            return value;
+        if (value == "THROUGHPUT")
```

Comment:
Let's consider introducing constants instead of typing strings for these critical values, I am worried one misspelling might cause a hard-to-detect bug one day and could be a headache.

Otherwise, LGTM!

Reply:
Thanks for your suggestion. Updated.